airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -334,14 +357,76 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return f"repos/{stream_slice['repository']}/labels"
 
 
-class Teams(GithubStream):
+class Organizations(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/orgs#get-an-organization
+    """
+
+    def __init__(self, organizations: List[str], **kwargs):
+        super(GithubStream, self).__init__(**kwargs)
+        self.organizations = organizations
+
+    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
+        for organization in self.organizations:
+            yield {"organization": organization}
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"orgs/{stream_slice['organization']}"
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        yield response.json()
+
+
+class Repositories(Organizations):
+    """
+    API docs: https://docs.github.com/en/rest/reference/repos#list-organization-repositories
+    """
+
+    fields_to_minimize = ("owner",)
```

Comment:
@tovbinm could we keep this un-minimized? or is it redundant with something else? IMO it's useful to keep this info here since the owner might not be a contributor present on any other stream in the `user` field

Reply:
Yea we can keep it un-minimized.

Reply:
I don't have enough access to push to this branch but one of you here can just update the schemas to refer to the shared `user` object schema.